plugins/OVS/OVS_plugin.py

```diff
@@ -1,4 +1,6 @@
+import json
 import os
+import uuid
 
 from src.python.fog05.interfaces.NetworkPlugin import NetworkPlugin, BridgeNotExistingException, \
```

Comment:
Replace the import without mentioning src.pyhton, should be 

```
from fog05.interface.NetworkPlugin import NetworkPlugin, BridgeNotExistingException,InterfaceNotExistingException 
```
Because after installation src.python will not be part of the path